src/cocoa/toga_cocoa/command.py

```diff
@@ -1,9 +1,7 @@
 
 class Command:
     def __init__(self, interface):
-        self.interface = interface
-        self.native = []
+        pass
```

Comment:
The `native` representation is still needed to support toolbar items; without it, tutorial 2 crashes.